Functions/xASL_test_BIDSConversion.m

```diff
@@ -81,10 +81,10 @@ function xASL_test_BIDSConversion(baseDirImport, baseDirReference, bImport, bCom
 			case 'Philips_PCASL_3DGRASE_R5.4_TopUp'
                 DirASL = fullfile(baseDirImport, flavorList{iFlavor}, 'analysis', 'Sub1', 'ASL_1');
                 
-				xASL_Move(fullfile(DirASL, 'M0_601_1.nii'), fullfile(DirASL, 'M0.nii'), 1);
-				xASL_Move(fullfile(DirASL, 'M0_601_1.json'), fullfile(DirASL, 'M0.json'), 1);
-				xASL_Move(fullfile(DirASL, 'M0_701_1.nii'), fullfile(DirASL, 'M0PERev.nii'), 1);
-				xASL_Move(fullfile(DirASL, 'M0_701_1.json'), fullfile(DirASL, 'M0PERev.json'), 1);
+				xASL_Move(fullfile(DirASL, 'M0_601_00001.nii'), fullfile(DirASL, 'M0.nii'), 1);
```

Comment:
@jan-petr 
These 601 701 look study-specific, like the 6th or 7th sequence that was scanned? That varies between studies/data sets, so this should probably be more flexible.